README.md

```diff
@@ -1,7 +1,6 @@
 # Add Badge
 
-A set of node commands to add an overlay badge, inset to the non-transparent area. Meant for PNG app icons with equal
-insets.
+Node commands to add an overlay badge, inset to the non-transparent area.
 
 | ![](https://github.com/sourcetoad/add-badge/raw/master/samples/output/ic_launcher-xxxhdpi.png) | ![](https://github.com/sourcetoad/add-badge/raw/master/samples/output/ic_launcher_round-xxxhdpi.png) |
```

Comment:
They are showing as the old ones because they are an absolute link to `master`, they should fix themselves if this gets merged since the file name was not changed for these sizes/variants.  They were absolute links so things would load fine on npmjs but I did not test that it was still needed, I just know I had issues with relative images there in the past.